src/robusta/integrations/discord/sender.py

```diff
@@ -38,6 +41,23 @@
 BLANK_CHAR = "\u200b"  # Discord does not allow us to send empty strings, so we use blank char instead
 
 
+class FindingStatus(Enum):
```

Comment:
This one also looks similar to base.py
Just the hex color changes

Is there a way to make some reuse of the FindingStatus from base.py?

Reply:
Sure.

Reply:
done